cmd/copyEnumeratorInit.go

```diff
@@ -233,6 +233,14 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 		srcRelPath := cca.makeEscapedRelativePath(true, isDestDir, object)
 		dstRelPath := cca.makeEscapedRelativePath(false, isDestDir, object)
 
+		if runtime.GOOS == "windows" && strings.HasPrefix(dstRelPath, "/%5C") {
```

Comment:
Sorry I'm a bit confused here. Could you please add a comment to explain what is coming in, and what are we doing to it? An example would be best.

Reply:
Added comments for better understanding.

So basically when a user tries to copy whole volume in windows, "//" gets added to the name of all the files/directories in the destination. "D:\\file.txt" will get uploaded as "\\file.txt" and "D:" will not be created. So I've tried to fix it by getting the volume name from source and then stripping "/%5C" from dstRelPath to create valid path.